src/lwc/geFormFieldLookup/geFormFieldLookup.js

```diff
@@ -8,22 +9,57 @@ export default class GeFormFieldLookup extends LightningElement {
     @api fieldApiName;
     @api objectApiName;
     @api displayValue = '';
+    @api defaultValue;
     @api label;
     @api required;
     @api id; // unique identifier for this field, used mainly for accessibility
 
 
     @track options = [];
-    @track targetObjectInfo;
     @track value;
     @track targetObjectApiName;
+    @track queryFields;
+    @track valid = true;
 
     /**
      * Retrieve information about the object the lookup points to.
      */
     @wire(getObjectInfo, { objectApiName: '$targetObjectApiName' })
     wiredTargetObjectInfo(response) {
-        this.targetObjectInfo = response;
+        if(response.data) {
+            this.targetObjectInfo = response;
+            this.queryFields = this.getQueryFields();
+        }
+    }
+
+    @wire(getRecord, { recordId: '$defaultValue', fields: '$queryFields'})
```

Comment:
You could use optionalFields instead of fields, so we don't get an error if the user doesn't have access to some of the fields.

Reply:
I think in this case we are good without optionalFields, we're grabbing data that's needed for our Lookup component to function. If we don't have access to Id or Name, I think it's ok that the lookup component doesn't work because those fields are essential to it.